MN/Algoritmi/LU/Crout/factorCrout.m

```diff
@@ -0,0 +1,13 @@
+function [L U] = factorCrout(A)
```

Comment:
```matlab
function [L, U, x] = Crout(A, b)
```
Crout rezolva un sistem.
Punem virgula in vectorul returnat.
Numim cu litera mare la inceput (CUM AM SCRIS IN README).